dimod/serialization/fileview.py

```diff
@@ -208,8 +255,8 @@ def load(fp, cls=None):
 
     """
 
-    if isinstance(fp, (bytes, bytearray)):
-        fp = io.BytesIO(fp)
+    if isinstance(fp, (bytes, bytearray, memoryview)):
```

Comment:
Still going to use the custom class with `bytes`?

Reply:
No harm and consistency is good I think

Reply:
:+1: